client/src/components/overview/GalleryView.jsx

```diff
@@ -124,19 +124,82 @@ function Overview(props) {
         console.log(err);
       })
   }, []);
-  return (
-    // The grid
-    <StyledOverviewGrid>
-      <OverviewCarousel photos={styles[styleIndex].photos}/>
-      <OverViewStars stars={rating}/>
-      <OverViewName name={productInfo.name} category={productInfo.category}/>
-      <OverViewPrice price={styles[styleIndex]}/>
-      <OverViewSelector styles={styles} setStyles={setStyleIndex} styleIndex={styleIndex}/>
-      <OverViewForm styles={styles[styleIndex]}/>
-      <OverviewDescription description={productInfo.description}/>
-      <OverviewFacts facts={productInfo.features}/>
-    </StyledOverviewGrid>
-  )
+
+  const isInitialMount = useRef(true);
+  useEffect(() => {
+    if (isInitialMount.current) {
+     isInitialMount.current = false;
+    } else {
+      var zoomBoxes = document.querySelectorAll('.detail-view');
+
+      // Extract the URL
+      zoomBoxes.forEach(function(image) {
+        var imageCss = window.getComputedStyle(image, false),
+          imageUrl = imageCss.backgroundImage
+                            .slice(4, -1).replace(/['"]/g, '');
```

Comment:
nice use of regex!

Reply:
Yup I didn't write that. That was part of what I had copied.